.ci.yaml

```diff
@@ -4409,6 +4409,10 @@ targets:
     properties:
       tags: >
         ["devicelab","hostonly"]
+      dependencies: >-
```

Comment:
You can remove `bringup: true` to run it in presubmit to see if this fixes it before landing

Reply:
thanks! that worked great :D